### PR TITLE
Stories rewok: VaAppBar

### PR DESCRIPTION
**Pull request by Roman4437**
AppBar stories

### CODE REVIEW

File: packages/ui/src/components/va-app-bar/VaAppBar.stories.ts [hunk 1]
```diff
@@ -1,12 +1,132 @@
...
+        id="target1" 
+        class="pb-[500px] overflow-auto"
+      >
+        {{ $vb.lorem(2500) }}
+      </div>
+    </div>
+  `,
+})
+
+export const ShadowOnScroll = () => ({
```
Comment (asvae) [inline packages/ui/src/components/va-app-bar/VaAppBar.stories.ts:76]: Would also be nice to have interaction demo.

File: packages/ui/src/components/va-app-bar/VaAppBar.stories.ts [hunk 2]
```diff
@@ -1,12 +1,132 @@
...
+        id="target2" 
+        class="pb-[500px] overflow-auto"
+      >
+        {{ $vb.lorem(2500) }}
+      </div>
+    </div>
+  `,
+})
+
+export const ShadowColor = () => ({
```
Comment (asvae) [inline packages/ui/src/components/va-app-bar/VaAppBar.stories.ts:96]: Can we scroll this by default somehow? I'd like visual test to capture shadow color.

**Review by asvae — CHANGES_REQUESTED** [review]
Pretty good.

Let's work on this some more.

File: packages/ui/src/components/va-app-bar/VaAppBar.stories.ts (hunk 2 above)
Comment (Roman4437) [reply packages/ui/src/components/va-app-bar/VaAppBar.stories.ts:121]: it can't be helped

**Review by asvae — APPROVED** [review]

**Pull request merged**